packages/web3torrent/src/components/torrent-info/wires-list/WiresList.tsx

```diff
@@ -0,0 +1,56 @@
+import prettier from 'prettier-bytes';
+import React from 'react';
+import {Torrent} from '../../../types';
+import './WiresList.scss';
+import {ChannelState} from '../../../clients/payment-channel-client';
+import {PaidStreamingWire} from '../../../library/types';
+
+export type UploadInfoProps = {
+  torrent: Torrent;
+  channelCache: Record<string, ChannelState>;
+  channelIds: string[];
+  peerType: 'seeder' | 'leecher';
+};
```

Comment:
Nitpick: Based on your usage, the params can be distilled into just `{wires, channels, peerType}`.  It adds a little work for the call site, but the code in this component is cleaner because it does less work on channel id filtering and deconstructing `torrent`.